src/NodeAnalyzer/TestsNodeAnalyzer.php

```diff
@@ -58,6 +58,16 @@ public function isTestClassMethod(ClassMethod $classMethod): bool
             return true;
         }
 
+        foreach ($classMethod->getAttrGroups() as $attrGroup) {
+            $hasTestAttribute = array_any(
```

Comment:
`array_any` seems only on php 8.0+, we don't have `array_any` downgrade rule yet so should be use different logic for now as we downgrade to php 7.4

Reply:
ah thanks for noticing. I've updated the logic to the version I started out with - before PhpStorm suggested to use `array_any` ;)